pdr_backend/accuracy/app.py

```diff
@@ -77,13 +79,18 @@ def process_single_slot(
         int(slot.ID.split("-")[1])
     )  # Using dot notation for attribute access
 
+    today = 0
```

Comment:
these variables are named like they represent times, but they are ints. What are they supposed to hold?